pkg/router/httpTriggers.go

```diff
@@ -143,13 +143,23 @@ func versionHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func NotFoundHandler(ts *HTTPTriggerSet) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ts.logger.Info("404 Not Found", zap.String("req_method", r.Method), zap.String("url_path", r.URL.Path), zap.String("req_address", r.RemoteAddr), zap.String("Time", time.Now().Format(time.RFC3339)), zap.Int("route_table_size", len(ts.functions)))
```

Comment:
<img width="1551" alt="CleanShot 2024-05-23 at 14 19 25@2x" src="https://github.com/gadget-inc/fission/assets/30190842/7c65908e-05e8-4249-a9b9-b592d8941342">

Note the route_table_size matches the number of functions so I'm confident this will provide good insight. (I snake_cased it in this change)